### ISSUE TITLE
Mesh Animation not working properly after build. 

### ISSUE THREAD

**Issue by ErsjanKeri** [plouc/nivo#1475]
a list of my dependencies: 
```
"dependencies": {
    "-": "^0.0.1",
    "@nivo/bar": "^0.67.0",
    "@nivo/chord": "^0.67.0",
    "@nivo/core": "^0.67.0",
    "@nivo/line": "^0.67.0",
    "@nivo/tooltip": "^0.67.0",
    "@react-spring/web": "^9.0.0",
    "@testing-library/jest-dom": "^5.11.9",
    "@testing-library/react": "^11.2.5",
    "@testing-library/user-event": "^12.8.3",
    "antd": "^4.14.1",
    "axios": "^0.21.1",
    "core-js": "3",
    "d3": "^6.6.2",
    "install-peers": "^1.0.3",
    "konva": "^7.2.5",
    "nivo": "^0.31.0",
    "prop-types": "^15.7.2",
    "react": "17.0.0",
    "react-dom": "^17.0.1",
    "react-konva": "^17.0.2-0",
    "react-motion": "^0.5.2",
    "react-redux": "^7.2.3",
    "react-router-dom": "^5.2.0",
    "react-scripts": "4.0.3",
    "react-spring": "^9.0.0",
    "redux-devtools-extension": "^2.13.9",
    "redux-thunk": "^2.3.0",
    "save-dev": "^0.0.1-security",
    "three": "^0.127.0",
    "web-vitals": "^1.1.1",
    "yarn": "^1.22.10",
    "zdog": "^1.1.2"
  },
```
The mesh animation does not work properly after build is finished, it freezes and does not run smoothly, but when I run on local server everything works perfectly fine.
Am I missing any dependencies for it to run properly? I have been looking up for days but I cannot find a solution to this problem.

**Comment by katlavan** [comment]
The same issue after the update. Labels, lines sticks after resize

**Comment by wyze** [comment]
This was fixed via #1480.
